File: 프로그래머스/lv0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.js
function solution(n) {
    var answer = 0;
    if (parseInt(n / 7) > 1) {
        if (n % 7 == 0) {
            answer = n / 7
        } else {
            answer = parseInt(n / 7) + 1
        }
    } else {
        answer = 1
    }
    return answer;
}

File: 프로그래머스/0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.swift
import Foundation

func solution(_ n:Int) -> Int {
    var answer: Int = 1
    if n < 8 { 
        answer = 1
    } else {
        if n % 7 == 0 {
            answer = n / 7
        } else {
            answer = n / 7 + 1
        }
    }
    return answer
}